ci_group/revolve2/ci_group/genotypes/cppnwin/modular_robot/v1/_body_develop.py

```diff
@@ -84,7 +76,7 @@ def __evaluate_cppn(
     :param body_net: The CPPN network.
     :param position: Position of the module.
     :param chain_length: Tree distance of the module from the core.
-    :returns: (module type, orientation)
+    :returns: (module type, rotation_index)
     """
     body_net.Input(
```

Comment:
can you add an assert that position actually consists of integers? that would give me more confidence. the numpy typing is so bad